### PR TITLE
Load transfer templates from config

### PR DESCRIPTION
**Pull request by yoldas**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: config/default_records/transfer_templates/bioscan_transfer_templates.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# These transfer templates are associated with the Bioscan pipelines.
```
Comment (andrewsparkes) [inline config/default_records/transfer_templates/bioscan_transfer_templates.yml:1]: It's only the '384 plate to tube' one that is associated with Bioscan, should probably leave all the others in seeds as we don't know what they are used for?

File: spec/data/record_loader/transfer_templates/bioscan_transfer_templates.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# These transfer templates are associated with the Bioscan pipelines.
```
[the remark this answers is not in the document]
Comment (andrewsparkes) [reply spec/data/record_loader/transfer_templates/bioscan_transfer_templates.yml:1]: but this test could use something generic, and not name the file with bioscan_

File: config/default_records/transfer_templates/bioscan_transfer_templates.yml (hunk 1 above)
Comment (yoldas) [reply config/default_records/transfer_templates/bioscan_transfer_templates.yml:1]: I have removed the others from the config.

Comment (yoldas) [reply config/default_records/transfer_templates/bioscan_transfer_templates.yml:1]: We have now only "384 plate to tube" in the bioscan_transfer_templates.yml file .

**Review by andrewsparkes — APPROVED** [review]

**Pull request merged**